apps/gauzy/src/app/pages/integrations/components/integration-list/list.component.html

```
@@ -36,86 +40,93 @@ <h4>
+													nbButton
+													size="tiny"
+													status="basic"
+													[disabled]="!integration.isActive"
+												>
+													<nb-icon class="mr-1" icon="eye-outline"></nb-icon>
+													{{ 'BUTTONS.VIEW' | translate }}
+												</button>
+											</ng-template>
+											<ng-template [ngxPermissionsOnly]="['INTEGRATION_DELETE']">
+												<button
+													(click)="deleteIntegration(integration)"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)